library/core/src/intrinsics.rs

```diff
@@ -1388,6 +1388,21 @@ extern "rust-intrinsic" {
     /// [`f64::round`](../../std/primitive.f64.html#method.round)
     pub fn roundf64(x: f64) -> f64;
 
+    /// Returns the nearest integer to an `f32`. Rounds half-way to the nearest even
+    /// integer.
+    ///
+    /// The stabilized version of this intrinsic is
+    /// [`f32::round`](../../std/primitive.f32.html#method.round)
```

Comment:
This links to `round`, not `round_to_even`.

Reply:
Dito. I have forgotten to update the docstring here. Let me fix that.